fileHub/js-app/tests/app/services/tests-router.js

```diff
@@ -0,0 +1,66 @@
+import {RoutingConfiguration} from '../../../app/services/routing-configuration.js';
+import {Router} from '../../../app/services/router.js';
+
+const {module, test} = QUnit;
+const DEFAULT_ROUTE = 'route';
+const REGISTER_ROUTE = 'register';
+const ERROR_ROUTE = '404';
+
+module('Router service', (hooks) => {
+  test('Should show default page when hash is correctly', (assert) => {
+    const testWindow = {
+      location: {hash: '#register'},
+
+      addEventListener(event, init) {
+        init();
+      },
+    };
+
+    const configuration = getConfig(assert);
+    new Router(configuration, testWindow);
+    assert.verifySteps([REGISTER_ROUTE, REGISTER_ROUTE], 'Should route to default page.');
+  });
+
+  test('Should show default page when hash is empty', (assert) => {
+    const testWindow = {
+      location: {hash: '#'},
+
+      addEventListener(event, init) {
+        init();
+      },
+    };
+
+    const configuration = getConfig(assert);
+    new Router(configuration, testWindow);
+    assert.verifySteps([DEFAULT_ROUTE, DEFAULT_ROUTE], 'Should route to default page.');
```

Comment:
Please checck location.hash.

Reply:
Also please check that router works correctly when hash is changed (It should render a page according to the new hash)